js/app.js

```diff
@@ -8,6 +8,163 @@ const menuBody = document.getElementById('mobile-menu-body');
+    title: 'CANOPY',
+    category: 'Back End Dev',
+    languages: ['html', 'css', 'ruby on rails'],
+    images: ['tonic-2-project.jpg', 'Snapshoot-Portfolio-3.png'],
+  },
+};
+
+function modalPopup(data) {
+  menuBody.style.overflow = 'hidden';
+  tonicProjectModal.style.display = 'block';
+  tonicProjectModal.innerHTML += `
+    <div class='tonic-project-modal-content' id='tonic-project-modal'>
+    <div class='tonic-project-content'>
+        <div class='tonic-project-modal-header'>
+            <h2 class='tonic-h2'>${data[0]}</h2>
+            <span class='tonic-project-close' onClick="document.getElementById('modal-div').style.display = 'none';
+            document.getElementById('mobile-menu-body').style.overflow ='visible';">&times;</span>
+        </div>      
+        <ul class='works-headers tonic-project-modal-header-techs'>
+            <li class='title-header' style='margin-right: 50px;'>${data[1]}</li>
+            <li><img src='images/works-dot.PNG' style='margin-right: 10px;' class='dot-separator' alt='Dot separator'></li>
+            <li class='title-header-general'  style='margin-right: 23px;'>${data[2]}</li>
+            <li><img src='images/works-dot.PNG' style='margin-right: 13px;' class='dot-separator' alt='Dot separator'></li>
+            <li class='title-header-general'>2015</li>
+        </ul>
+        <div class='tonic-project-modal-body'>
+            <figure class='img-wide-screen-modal'>
+                <img src='images/${data[5][0]}' class='works-image style-image small-screen-view' alt='Tonic App Screenshot'>
+                <img src='images/${data[5][1]}' class='wide-screen-view modal-pic-full' alt='Tonic project image Screenshot'>
+            </figure>
+        </div>
```

Comment:
- [ ] Please make sure the project image in the model contains more width to match the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A546) design.

### Your design
![image](https://user-images.githubusercontent.com/83803414/195917944-4e0eb711-5657-4563-9cf7-00061eea38e4.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/195917913-eea85edb-a615-4f33-be30-3ce5e69d2e1d.png)